lib/src/realm_property.dart

```diff
@@ -49,6 +49,9 @@ class SchemaProperty {
   /// Indicates that the property should be persisted under a different name
   final String? mapTo;
 
+  /// @nodoc
+  String get internalName => mapTo ?? name;
```

Comment:
```suggestion
  final String? _mapTo;

  /// @nodoc
  String get mapTo => _mapTo ?? name;
```
might also be a better option, but I'm okay with `internalName`